lib/sensu/server/tessen.rb

```diff
@@ -0,0 +1,165 @@
+require "em-http-request"
+require "sensu/constants"
+
+module Sensu
+  module Server
+    class Tessen
+      attr_accessor :settings, :logger, :redis, :options
+      attr_reader :timers
+
+      # Create a new instance of Tessen. The instance variable
+      # `@timers` is use to track EventMachine timers for
+      # stopping/shutdown.
+      #
+      # @param options [Hash] containing the Sensu server Settings,
+      #   Logger, and Redis connection.
+      def initialize(options={})
+        @timers = []
+        @settings = options[:settings]
+        @logger = options[:logger]
+        @redis = options[:redis]
+        @options = @settings.to_hash.fetch(:tessen, {})
+      end
+
+      # Determine if Tessen is enabled (opt-out).
+      #
+      # @return [TrueClass, FalseClass]
+      def enabled?
+        @options[:enabled] != false
```

Comment:
I feel like this is a breaking change and should be opt in for sensu 1.x and in sensu 2.x you can change the default as that is where all breaking changes should be made. I think it should be `@options[:enabled] || false` for it's initial release. We can always have the various CM tools change their defaults in a major version of the cookbook, modules, playbooks,formulas but if you want to avoid a major bump for sensu ruby and have committed to semver this is how it has to be. Also keep in mind that your data will be horribly skewed if its opt out default as it will include local development, ci, etc.